### PR TITLE
Update legal agreement pages for a11y

### PR DESCRIPTION
**Pull request by jessieay**
* Fix invalid HTML as detected by WAVE
* Fix contrast issue
* Contrast issue brought up issue with our overrides, which was that we
load our overrides last in the manifest file so that they take
precedent, which means that our variable overrides were not loaded until
after all of the base styles. Now, we are loading our variable overrides
above the base styles so that the base styles that use variables are
using our overrides rather than the original colors.
* [Finishes #153764205]
* [Finishes #153764168]

screen shot 2017-12-19 at 9 14 25 am

### CODE REVIEW

File: app/views/shared/_terms.html.erb [hunk 1]
```diff
@@ -724,20 +774,17 @@ meals.</p>
...
   <li>Applying for Assistance: <a href="http://www.michigan.gov/dhs-applicationprocess">www.michigan.gov/dhs-applicationprocess</a></li>
-  <li><a href="http://www.michigan.gov/dhs-applicationprocess"></a></li>
   <li>Cash Assistance: <a href="http://www.michigan.gov/dhs-cash">www.michigan.gov/dhs-cash</a></li>
-  <li ><a href="http://www.michigan.gov/dhs-cash"></a></li>
-    <li >Emergency Services: <a href="http://www.michigan.gov/dhs-ser">www.michigan.gov/dhs-ser</a></li><li
-><a href="http://www.michigan.gov/dhs-ser"></a></li><li
-    >Food Assistance: <a href="http://www.michigan.gov/foodstamps">www.michigan.gov/foodstamps"</a></li>
-    <li >Medical Services: <a href="http://www.michigan.gov/dhs-medical">www.michigan.gov/dhs-medical</a></li>
+  <li>Emergency Services: <a href="http://www.michigan.gov/dhs-ser">www.michigan.gov/dhs-ser</a></li>
+    <li>Food Assistance: <a href="http://www.michigan.gov/foodstamps">www.michigan.gov/foodstamps"</a></li>
```
Comment (luigi) [inline app/views/shared/_terms.html.erb]: Appears to have indented accidentally starting on this line. This line also has a dangling `"` before `</a>`.

File: app/views/shared/_terms.html.erb [hunk 2]
```diff
@@ -520,73 +520,123 @@ http://www.michigan.gov/disabilityresources/
...
+</p>
 
 <p>Resources:</p>
 
 <ul>
-  <li >Online at: <a href="http://www.michigan.gov/domesticviolence>www.michigan.gov/domesticviolence"</a></li>
-  <li >DHS Publication 859, Is Someone Hurting You or Your Children? (also available in Spanish) -</li>
+  <li>
+    Online at: <a
+      href="http://www.michigan.gov/domesticviolence">www.michigan.gov/domesticviolence"</a>
```
Comment (luigi) [inline app/views/shared/_terms.html.erb]: Dangling `"` before `</a>`.

File: app/views/shared/_terms.html.erb [hunk 3]
```diff
@@ -520,73 +520,123 @@ http://www.michigan.gov/disabilityresources/
...
+    DHS Publication 859, Is Someone Hurting You or Your Children? (also
+    available in Spanish)
+  </li>
 </ul>
 
 <ul>
-  <li>Online at: <a href="http://www.michigan.gov/dhs-publications>www.michigan.gov/dhs-publications"</a></li>
+  <li>
+    Online at: <a
+      href="http://www.michigan.gov/dhs-publications">www.michigan.gov/dhs-publications"</a>
```
Comment (luigi) [inline app/views/shared/_terms.html.erb]: Dangling `"` before `</a>`.

File: app/views/shared/_terms.html.erb (hunk 2 above)
Comment (jessieay) [reply app/views/shared/_terms.html.erb]: oh yeah that's in an attempt to make the lines shorter :)

File: app/views/shared/_terms.html.erb (hunk 3 above)
Comment (jessieay) [reply app/views/shared/_terms.html.erb]: same as before ;)

File: app/views/shared/_terms.html.erb (hunk 1 above)
Comment (jessieay) [reply app/views/shared/_terms.html.erb]: good catch!

fixed at a3e34ad0

File: app/views/shared/_terms.html.erb (hunk 2 above)
Comment (jessieay) [reply app/views/shared/_terms.html.erb]: woops, didnt read your comment correctly

fixed at 19db043f

**Pull request merged**